site/docs/get-started/quickstart.md

```diff
@@ -108,26 +108,46 @@ Run the following to install vault and the plugin in development mode:
     kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
     ```
 
+## Install data catalog
+
+Fybrik assumes the existence of a data catalog that contains the metadata and connection information for data assets. Fybrik currently supports:
+
+ 1. [OpenMetadata](https://open-metadata.org/): An open-source end-to-end metadata management solution that includes data discovery, governance, data quality, observability, collaboration, and lineage.
+ 2. [katalog](https://fybrik.io/dev/reference/katalog/): a data catalog that is included in Fybrik for evaluation purposes.
+
+In this guide we use the OpenMetadata data catalog.
+
+To use OpenMetadata, you can either use an existing deployment, or run the following commands to deploy OpenMetadata in kubernetes:
+```bash
+curl https://raw.githubusercontent.com/fybrik/fybrik/master/third_party/openmetadata/install_OM.sh | bash -
+```
+
+The installation of OpenMetadata could take a long time (around 20 minutes on a VM running [kind](https://kind.sigs.k8s.io/) Kubernetes).
+
+Running `make` installs OpenMetadata in the `open-metadata` namespace. To install OpenMetadata in another namespace, or to change the credentials of the different services used by OpenMetadata, edit the variables in the `Makefile.env` file.
```

Comment:
Perhaps add a link to the Makefile.env?

Reply:
> Running `make` installs OpenMetadata .... 

But a user doesn't run `make` he runs `curl` which runs  `install_OM.sh`. And the last one downloads the required files, runs `make` and removes the downloaded files.
Therefore,  the user doesn't have access to the `Makefile.env` file. 

One of possible solutions can be to add a "command" to the `install_OM.sh` file, e.g. `getFiles`, "cleanup", "install". If no command is provided, the script will do all of the operations (like now)

Reply:
Thanks. I will take care of it as part of https://github.com/fybrik/fybrik/pull/1791